openmdao/core/problem.py

```diff
@@ -428,7 +428,11 @@ def set_val(self, name, value, units=None, indices=None):
             Indices or slice to set to specified value.
         """
         model = self.model
-        conns = self._metadata['connections']
+        try:
+            conns = self._metadata['connections']
+        except AttributeError:
+            raise RuntimeError(f"{self.msginfo} '{name}': Cannot call set_val before setup.")
```

Comment:
looks like for uniformity, we always put the colon after the self.msginfo.  Look at the other error messages for example.